local-volume/provisioner/pkg/common/common.go

```diff
@@ -67,9 +71,9 @@ type UserConfig struct {
 // MountConfig stores a configuration for discoverying a specific storageclass
 type MountConfig struct {
 	// The hostpath directory
-	HostDir string `json:"hostDir"`
+	HostDir string `json:"hostDir" yaml:"hostDir"`
```

Comment:
Do we want to support both json and yaml encodings?

Reply:
I noticed we need both, as actually yaml function does something with json undercover.